CHANGELOG.md

```diff
@@ -17,20 +17,22 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/o1-labs/o1js/compare/3eef10d...HEAD)
 
+- [PR !2143]( https://github.com/o1-labs/o1js/pull/2143) Lazy mode for prover index computation
```

Comment:
the changelog seems to be a bit out of date, make sure to bring it up to date with main